manifests/init.pp

```diff
@@ -84,6 +84,9 @@
 # @param allowed_import_path
 #   Allowed paths that pulp can sync from using file:// protocol
 #
+# @param worker_count
+#   Number of pulpcore workers. Defaults to 8 or the number of CPU cores, whichever is smaller. Enabling more than 8 workers, even with additional CPU cores available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
```

Comment:
Mind wrapping the help text 1-2 lines so the file is not rendered really wide?

Reply:
Done, thanks